### PR TITLE
Jmrt changes

### PR DESCRIPTION
**Pull request by delarosatrevin**
### Proposed changes:
* Define a MODEL_ANGELO_ACTIVATION that does not depend on scipion/conda/envs (e.g cluster installations and loaded via `module load`
* Fixed var definitions using `_defineVar` to load values from the config
* Some PEP8 and other minor formatting
* Added validation to alert users that only one GPU can be used

### CODE REVIEW

File: modelangelo/protocols/protocol_model_angelo.py [hunk 1]
```diff
@@ -76,44 +71,41 @@ def _defineParams(self, form):
                       pointerClass="Sequence", allowsNull=True, important=True,
                       label='Protein sequences',
                       help="Include here one or more sequences to be modeled\n"
-                           "Leave empty is you want to use the model_no_seq option")
+                           "Leave empty is you want to use the model_no_seq "
```
Comment (pconesa) [inline modelangelo/protocols/protocol_model_angelo.py]: hehe, there was a typo already you missed : is -->if

**Review by pconesa — CHANGES_REQUESTED** [review]
Thanks for improving this.
Could you please:

correct the typo
add instructions in the README about the variables. There were already missing ;-).
and bump the version ?

**Comment by delarosatrevin** [comment]
Done!

**Review by pconesa — APPROVED** [review]
Good job!

**Pull request merged**